jenkins/Jenkinsfile.daily

```
@@ -0,0 +1,45 @@
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(
+      logRotator(
+        artifactDaysToKeepStr: '',
+        artifactNumToKeepStr: '',
+        daysToKeepStr: '',
+        numToKeepStr: '100'))
+  }
+
+  triggers {
+    // 6am UTC or 1am EST
+    cron '0 6 * * *'
+  }
+  
+  parameters {
+    booleanParam name: 'PUBLISH', defaultValue: true, description: 'Publish the build to S3 and sentry.'
+    booleanParam name: 'FORCE_BUILD_BINARIES', defaultValue: false, description: 'Force build binaries even if there are no changes, and even if they have already been built previously'
+    booleanParam name: 'FORCE_BUILD_DOCKER', defaultValue: false, description: 'Force build docker images even if there are no dockerfile changes'
+    string name: 'OS_FILTER', defaultValue: 'all', description: 'Pattern to limit builds by matching OS'
+    string name: 'ARCH_FILTER', defaultValue: 'all', description: 'Pattern to limit builds by matching ARCH'
+    string name: 'SLACK_CHANNEL', defaultValue: '#ide-builds', description: 'Slack channel to publish build message.', trim: true
+  }
+  
+  stages {
+    stage('Trigger Main Daily Build') {
```

A comment suggestion, just in case someone looking in this file might be expecting the release daily stage to be in here as well

```suggestion
  // Release branch daily builds are defined in Jenkinsfile.release
  stages {
    stage('Trigger Main Daily Build') {
```